packages/editor-ui/src/components/DeleteUserModal.test.ts

```diff
@@ -0,0 +1,143 @@
+import { createComponentRenderer } from '@/__tests__/render';
+import DeleteUserModal from './DeleteUserModal.vue';
+import { createTestingPinia } from '@pinia/testing';
+import { getDropdownItems } from '@/__tests__/utils';
+import { createProjectListItem } from '@/__tests__/data/projects';
+import { createUser } from '@/__tests__/data/users';
+
+import { DELETE_USER_MODAL_KEY } from '@/constants';
+import { ProjectTypes } from '@/types/projects.types';
+import userEvent from '@testing-library/user-event';
+import { useUsersStore } from '@/stores/users.store';
+
+const ModalStub = {
+	template: `
+		<div>
+			<slot name="header" />
+			<slot name="title" />
+			<slot name="content" />
+			<slot name="footer" />
+		</div>
+	`,
+};
+
+const loggedInUser = createUser();
+const invitedUser = createUser({ firstName: undefined });
+const user = createUser();
+
+const initialState = {
+	ui: {
+		modalsById: {
+			[DELETE_USER_MODAL_KEY]: {
+				open: true,
+			},
+		},
+		modalStack: [DELETE_USER_MODAL_KEY],
+	},
+	projects: {
+		projects: [
+			ProjectTypes.Personal,
+			ProjectTypes.Personal,
+			ProjectTypes.Team,
+			ProjectTypes.Team,
+		].map(createProjectListItem),
+	},
+	users: {
+		usersById: {
+			[loggedInUser.id]: loggedInUser,
+			[user.id]: user,
+			[invitedUser.id]: invitedUser,
+		},
+	},
+};
+
+const global = {
+	stubs: {
+		Modal: ModalStub,
+	},
+};
+
+describe('DeleteUserModal', () => {
+	it('should delete invited users', async () => {
+		const { getByTestId } = createComponentRenderer(DeleteUserModal)({
```

Comment:
We usually save the result of `createComponentRenderer(DeleteUserModal)` so it is not repeated, especially if it always gets the same argument.

e.g.: `const renderComponent = createComponentRenderer(DeleteUserModal)`
you can declare it before the first `describe` block